tests/bindings-framework-test/bindings-framework-test.csproj

```diff
@@ -80,9 +67,23 @@
   </ItemGroup>
   <ItemGroup>
     <TestLibrariesInput Include="..\..\tests\test-libraries\libframework.m" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XTest.framework" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XSharedObjectTest.framework" />
-    <TestLibrariesOutput Include="..\..\tests\test-libraries\.libs\ios-fat\XSharedARTest.framework" />
```

Comment:
You still need these three lines for the `BeforeBuild` target below to work properly.